state/store/logic.js

```diff
@@ -1,5 +1,7 @@
 import sessionService from '../session/operations';
+import fetchTrendingMoviesLogic from '../movies/service';
 
 export default [
-  ...sessionService
+  ...sessionService,
+  fetchTrendingMoviesLogic
```

Comment:
Also, export logics as array too, to have the same approach as for sessions

Reply:
Done